Delta/Emulation/GameViewController.swift

```diff
@@ -1070,6 +1087,71 @@ extension GameViewController
             emulatorCore.rate = emulatorCore.deltaCore.supportedRates.lowerBound
         }
     }
+    
+    func performStatusBarAction()
+    {
+        let enabled = ExperimentalFeatures.shared.showStatusBar.isStatusBarEnabled
+        ExperimentalFeatures.shared.showStatusBar.isStatusBarEnabled = !enabled
+        
+        if let pauseView = self.pauseViewController
+        {
+            pauseView.dismiss()
+        }
+        
+        self.setNeedsStatusBarAppearanceUpdate()
+    }
+    
+    func performScreenshotAction()
+    {
+        guard let snapshot = self.emulatorCore?.videoManager.snapshot() else { return }
+        
+        let scale = ExperimentalFeatures.shared.gameScreenshots.scale != nil ? ExperimentalFeatures.shared.gameScreenshots.scale?.rawValue : 1.0
```

Comment:
```suggestion
        let scale = ExperimentalFeatures.shared.gameScreenshots.scale?.rawValue ?? 1.0
```

This will result in a non-optional variable for `scale`, so you don't need to implicitly unwrap it every time you use it.